### DISCUSSION TITLE
Failing install sc-im on Ubuntu

### DISCUSSION THREAD

**Discussion by bluedogtwo** [Homebrew/discussions#1559] [Everyday usage]
make
prefix=/home/linuxbrew/.linuxbrew/Cellar/sc-im/0.8.1

yacc -d gram.y
make: yacc: Command not found
make: *** [Makefile:203: y.tab.c] Error 127

**Comment by SMillerDev** [comment]
This is by far not enough information to help you.

What are you trying to do?

**Reply by bluedogtwo** [reply]
I'm trying to install SC-IM with brew (brew install sc-im).

**Reply by SMillerDev** [reply]
And can you post the full output you get when you run `brew install sc-im`

**Reply by bluedogtwo** [reply]
:~$ brew install sc-im
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 34 formulae.

==> Downloading https://github.com/andmarti1424/sc-im/archive/v0.8.1.tar.gz
Already downloaded: /home/abc/.cache/Homebrew/downloads/6b51aecf58641f8baec2d3b1a250e97a9ef68cbdbd76302b0841eb52dcb58d4b--sc-im-0.8.1.tar.gz
==> make prefix=/home/linuxbrew/.linuxbrew/Cellar/sc-im/0.8.1
Last 15 lines from /home/abc/.cache/Homebrew/Logs/sc-im/01.make:
2021-05-27 13:31:25 +0000

make
prefix=/home/linuxbrew/.linuxbrew/Cellar/sc-im/0.8.1

yacc -d gram.y
make: yacc: Command not found
make: *** [Makefile:203: y.tab.c] Error 127

READ THIS: https://docs.brew.sh/Troubleshooting

**Reply by gromgit** [reply]
It's missing at least one dependency (`bison`). Stand by for a fix...

**Reply by gromgit** [reply]
Fixed in https://github.com/Homebrew/linuxbrew-core/pull/23346. @bluedogtwo, try your `brew install` again.
